src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper/config.jelly

```diff
@@ -35,6 +21,20 @@
                 value="${instance.info.groovyScriptContent}"/>
     </f:entry>
 
+    <f:entry field="propertiesFilePath"
```

Comment:
As a summary of the discussion, I still think it should be moved to another PR:
1) It is completely unrelated to the PR
2) The change impacts the usability and UX IMO, because it changes the original layout